entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+cp -r /source/* /usr/src/app/
+
+exec $@
+
```

Comment:
Can this script be inlined into the `Dockerfile` (less clutter)? Also `#!/bin/sh` is sufficient here.